Formula/container-structure-test.rb

```diff
@@ -0,0 +1,25 @@
+class ContainerStructureTest < Formula
+  desc "Validate the structure of your container images"
+  homepage "https://github.com/GoogleContainerTools/container-structure-test"
+  url "https://github.com/GoogleContainerTools/container-structure-test.git",
+      :tag      => "v1.8.0",
+      :revision => "19abf36d1451cb27f8e0f5ec8260815c73184bd4"
+  head "https://github.com/GoogleContainerTools/container-structure-test.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
```

Comment:
See https://github.com/Homebrew/homebrew-core/issues/47627, we want to get rid of GOPATH usage

Reply:
The project isn't yet convert to a Go module. Can this wait until the project is migrated? I will open an issue on that project.

Reply:
@AndiDog FYI: https://github.com/GoogleContainerTools/container-structure-test/pull/246

Reply:
Please release a new version and I'll happily incorporate it into the formula

Reply:
Looks like there is a new RC, v1.9.0. I think we can bump to latest later.